```
      $BUILD_PREFIX/opt/emsdk/upstream/emscripten/cache/sysroot/include/c++/v1/__config:688:49: note: expanded from macro '_LIBCPP_DEPRECATED'
        688 | #      define _LIBCPP_DEPRECATED __attribute__((__deprecated__))
            |                                                 ^
      sage/symbolic/ginac/inifcns_orthopoly.cpp:23:10: fatal error: 'flint/fmpz_poly.h' file not found
         23 | #include "flint/fmpz_poly.h"
            |          ^~~~~~~~~~~~~~~~~~~
      3 warnings and 1 error generated.
      error: command '$BUILD_PREFIX/bin/em++' failed with exit code 1
      [end of output]
```

--- a/recipes/recipes_emscripten/passagemath-gsl/recipe.yaml
+++ b/recipes/recipes_emscripten/passagemath-gsl/recipe.yaml
@@ -1,0 +1,74 @@
+context:
+  version: 10.8.5
+
+package:
+  name: passagemath-gsl
+  version: ${{ version }}
+
+source:
+- url: https://pypi.org/packages/source/p/passagemath-gsl/passagemath_gsl-${{ version }}.tar.gz
+  sha256: 6280e538e314a60a8a911c9b71ea6776283cec81ac39f17bde7da6235d72b01a
+
+build:
+  script: ${{ PYTHON }} -m pip install .
+  number: 0
+
+requirements:
+  build:
+  - python
+  - crossenv >=1.2
+  - cross-python_emscripten-wasm32
+  - ${{ compiler("c") }}
+  - ${{ compiler("cxx") }}
+  - pip
+  - passagemath-setup
+  - passagemath-environment
+  - cython
+  - cysignals
+  - numpy
+  - pkgconfig
+  - passagemath-abi == ${{ version }}
+  host:
+  - python
+  - gsl
+  - gmp
+  - memory-allocator
+  - numpy
+  - passagemath-categories
+  - passagemath-modules
+  run:
+  - python
+  - cysignals
+  - memory-allocator
+  - numpy
+  - passagemath-categories
+  - passagemath-environment
+  - passagemath-modules
+
+tests:
+- script: pytester
+  files:
+    recipe:
+    - test_passagemath_gsl.py
+  requirements:
+    build:
+    - pytester
+    run:
+    - pytester-run
+    - passagemath-repl
+
+about:
+  license: GPL-2.0-or-later
+  license_file: README.rst
+  summary: 'passagemath: Numerics with the GNU Scientific Library'
+  description: |
+    Distribution of a part of the Sage library.
+    It provides modules that depend on the GNU Scientific Library.
+
+extra:
+  emscripten_tests:
+    python:
+      pytest_files:
+      - test_passagemath_gsl.py
+  recipe-maintainers:
+  - mkoeppe

--- a/recipes/recipes_emscripten/passagemath-gsl/test_passagemath_gsl.py
+++ b/recipes/recipes_emscripten/passagemath-gsl/test_passagemath_gsl.py
@@ -1,0 +1,5 @@
+import pytest
+
+
+def test_import_passagemath_gsl():
+    import passagemath_gsl

--- a/recipes/recipes_emscripten/passagemath-symbolics/recipe.yaml
+++ b/recipes/recipes_emscripten/passagemath-symbolics/recipe.yaml
@@ -10,7 +10,9 @@ source:
   sha256: 525b9a830e3af814702382751256ebda33d1da7e9515e5b4962a405d719e6809
 
 build:
-  script: ${{ PYTHON }} -m pip install .
+  script: |
+    export LDFLAGS="$LDFLAGS -Wl,--allow-multiple-definition"
+    ${{ PYTHON }} -m pip install .
   number: 0
 
 requirements:

--- a/recipes/recipes_emscripten/passagemath-symbolics/recipe.yaml
+++ b/recipes/recipes_emscripten/passagemath-symbolics/recipe.yaml
@@ -34,6 +34,8 @@ requirements:
   - mpfr
   - mpc
   - gsl
+  - libflint
+  - ntl
   - singular
   - numpy
   - cysignals

